api/types.proto

```diff
@@ -489,12 +489,17 @@ message NetworkAttachmentConfig {
 	// Target specifies the target network for attachment. This value must be a
 	// network ID.
 	string target = 1;
+	
 	// Aliases specifies a list of discoverable alternate names for the service on this Target.
 	repeated string aliases = 2;
+	
 	// Addresses specifies a list of ipv4 and ipv6 addresses
 	// preferred. If these addresses are not available then the
 	// attachment might fail.
 	repeated string addresses = 3;
+	
+	//Driver options is a map of driver specific options for the network target
```

Comment:
@stevvooe: I'm not sure I understand the question, but this is the definition of `Driver`:

```
// Driver is a generic driver type to be used throughout the API. For now, a
// driver is simply a name and set of options. The field contents depend on the
// target use case and driver application. For example, a network driver may
// have different rules than a volume driver.
message Driver {
        string name = 1;
        map <string, string> options = 2;
}
```